tools/proteomiqon_proteininference/proteomiqon_proteininference.xml

```diff
@@ -0,0 +1,267 @@
+<tool id="proteomiqon_proteininference" name="ProteomIQon ProteinInference" version="@VERSION@" profile="20.05">
+    <description>
+        uses a list of identified peptides to deduce proteins that explain their presence in the sample.
```

Comment:
can you make this more crisp/shorter?

Reply:
I'll take a look!

Reply:
How about: "uses identified peptides to deduce proteins explaining their presence in the sample."

Reply:
"uses identified peptides to infere proteins" ?